README.md

```diff
@@ -92,3 +92,27 @@ Best practice for managing a pull request
 
      ```make gh-publish```
 
+
+Use docker to build and preview the website
+--------------------------------------------
+To build CNERG website using docker you can use the cnerg container containing
+all the required dependencies to build the website cnerg/cenrg_XXXX.
+
+1. go into your folder hosting cnerg website sources
+
+2. run the docker container: 
+
+    ```docker run -v $PWD:/local_drive -it baaaaam/cnerg```
```

Comment:
baaaaaam --> username ?

Reply:
it has to be cnerg/something but I don't know what should be the name of the docker container yet